src/grz_cli/models/config.py

```diff
@@ -85,6 +85,11 @@ class S3Options(StrictBaseModel):
     Defaults to None.
     """
 
+    multipart_chunksize: int = 256 * 1024**2
```

Comment:
Is the spacing intentional to emphasize 256 * (1024^2)?

Reply:
Yes indeed.